pkg/deploy/lattice/access_log_subscription_synthesizer_test.go

```diff
@@ -78,4 +79,62 @@ func TestSynthesizeAccessLogSubscription(t *testing.T) {
 		err := synthesizer.Synthesize(ctx)
 		assert.NotNil(t, err)
 	})
+
+	t.Run("SpecIsDeleted_DeletesAccessLogSubscription", func(t *testing.T) {
+		ctx, c, mockManager, k8sClient, builder := setup()
+		defer c.Finish()
```

Comment:
can you change this setup too as you did above? can be in another PR

Reply:
Yup, fixed next commit!